CMakeLists.txt

```diff
@@ -17,13 +17,7 @@ if("${isSystemDir}" STREQUAL "-1")
 endif()
 
 # C++11
-if(CMAKE_VERSION VERSION_LESS "3.1")
-  set(CMAKE_CXX_FLAGS "--std=c++11 ${CMAKE_CXX_FLAGS}")
-else()
-  set(CMAKE_CXX_STANDARD 11)
-endif()
-
-set(CUDA_NVCC_FLAGS "--std=c++11")
+set(CMAKE_CXX_FLAGS "--std=c++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
Consider adding a comment so the next person who "helpfully" uses the "more correct, newer feature" avoids this problem ;)